### ISSUE TITLE
[locale.cons] Ambiguous argument in Throws for locale+name+category constructor

### ISSUE THREAD

**Issue by hubert-reinterpretcast** [cplusplus/draft#5292]
> `locale(const locale& other, const char* std_name, category);`

has

> _Throws:_ `runtime_error` if the argument is not valid, or is null.

There being three arguments, the statement is rather problematic.

It looks like a copy/paste from

> explicit locale(const char* std_name);

The conclusion seems to be that the statement should be changed to read:

> _Throws:_ `runtime_error` if `std_name` is not valid, or is null.

**Comment by hubert-reinterpretcast** [comment]
@jwakely, fyi.

**Comment by jensmaurer** [comment]
Is there any way that "category" can be invalid?

**Comment by jensmaurer** [comment]
>  Is the question whether the exception should also apply if the`category` value is not valid?

Yes. If there is no obvious answer, this looks like LWG material.

**Comment by hubert-reinterpretcast** [comment]
... and there's implementation divergence! LWG issue pending.
